src/main/java/mate/academy/spring/model/dto/response/OrderResponseDto.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.spring.model.dto.response;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import mate.academy.spring.model.Ticket;
+
+public class OrderResponseDto {
+    private Long id;
+    private List<Ticket> tickets;
+    private LocalDateTime orderDate;
```

Comment:
I can't see the correction...

![image](https://user-images.githubusercontent.com/20326858/210988938-dd7795df-9d44-46fa-a1b8-425de91291c3.png)

Reply:
I didn't understand the condition correctly, sorry